elytron/WFCORE-5120-client-default-ssl-context.adoc

```diff
@@ -0,0 +1,86 @@
+= WFCORE-5120 Automatic registration of client side / JVM wide default SSLContext
+:author:            Diana Vilkolak Krepinska
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This issue is about providing ability to register a client side JVM wide default SSLContext for the use of any libraries that support the use of the default SSL context.
+The loading of elytron client configuration should happen with minimal configuration from the user in cases where API is not used directly.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/ELY-2112[ELY-2112]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFCORE-4144[WFCORE-4144]
+* https://issues.redhat.com/browse/WFLY-13762[WFLY-13762]
+* https://issues.redhat.com/browse/WFCORE-5101[WFCORE-5101]
+
```

Comment:
I see that those issue are actually directly relating to a similar issue but in server side scope, i.e. https://issues.redhat.com/browse/WFCORE-4144.

Maybe those are actually related but I am wondering why the issues which are directly related to this one are not listed in here, i.e.:

* https://issues.redhat.com/browse/WFCORE-5120
* https://issues.redhat.com/browse/WFLY-14846